frame/system/src/lib.rs

```diff
@@ -367,9 +367,6 @@ decl_storage! {
 		/// the `EventIndex` then in case if the topic has the same contents on the next block
 		/// no notification will be triggered thus the event might be lost.
 		EventTopics get(fn event_topics): map hasher(blake2_256) T::Hash => Vec<(T::BlockNumber, EventIndex)>;
-
-		/// A bool to track if the runtime was upgraded last block.
-		pub RuntimeUpgraded: bool;
```

Comment:
One last nit is that we should use your new feature to remove this storage item. Otherwise, it is looking good to me 👍